powerline-themes.el

```diff
@@ -260,6 +260,158 @@
 			     (powerline-fill nil (powerline-width rhs))
 			     (powerline-render rhs)))))))
 
+(defface powerline-modified-face
+  '((((class color))
+     (:background "#FFA335" :foreground "black" :weight bold))
+    (t (:weight bold)))
+  "Face to fontify modified files."
```

Comment:
Can you please update these comments to say that they are only relevant to the leuven theme?